src/content/self-hosted/eks.mdx

```diff
@@ -83,8 +83,16 @@ license: 1234567890ABCD==
 subdomain: dev.example.com
 cluster:
   endpoint: "https://XXXXXX.gr7.us-west-2.eks.amazonaws.com"
+
+ingress-nginx:
+  controller:
+    service:
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
 ```
 
+Annotation `service.beta.kubernetes.io/aws-load-balancer-type: nlb` is used to tell AWS to create a [Network Load Balancer (NLB)](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) instead of the default [Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html) to expose the ingress controller. We recommend to use a NLB as it provides benefits like: websocket support, static IPs, and better performance. You can see a full comparasion between the load balancers [here](https://aws.amazon.com/elasticloadbalancing/features/).
```

Comment:
```suggestion
Annotation `service.beta.kubernetes.io/aws-load-balancer-type: nlb` is used to tell AWS to create a [Network Load Balancer (NLB)](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) instead of the default [Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html) to expose the ingress controller. We recommend to use a NLB as it provides benefits like: websocket support, static IPs, and better performance. You can see a full comparasion between the load balancers [here](https://aws.amazon.com/elasticloadbalancing/features/).
```